File: VB/MyWinExplorerView/MyWinExplorerViewInfoRegistrator.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid.Views.Base.Handler
Imports DevExpress.XtraGrid.Views.Base.ViewInfo

Namespace MyXtraGrid
    Public Class MyWinExplorerViewInfoRegistrator
        Inherits WinExplorerViewInfoRegistrator

        Public Overrides ReadOnly Property ViewName() As String
            Get
                Return "MyWinExplorerView"
            End Get
        End Property
        Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
            Return New MyWinExplorerView()
        End Function
        Public Overrides Function CreateViewInfo(ByVal view As BaseView) As BaseViewInfo
            Return New MyWinExplorerViewInfo(TryCast(view, MyWinExplorerView))
        End Function
        Public Overrides Function CreateHandler(ByVal view As BaseView) As BaseViewHandler
            Return New MyWinExplorerViewHandler(TryCast(view, MyWinExplorerView))
        End Function
    End Class
End Namespace

File: CS/MyWinExplorerView/MyWinExplorerViewInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid.Views.Base.Handler;
using DevExpress.XtraGrid.Views.Base.ViewInfo;

namespace MyXtraGrid
{
    public class MyWinExplorerViewInfoRegistrator : WinExplorerViewInfoRegistrator
    {
        public override string ViewName { get { return "MyWinExplorerView"; } }
        public override BaseView CreateView(GridControl grid) { return new MyWinExplorerView(); }
        public override BaseViewInfo CreateViewInfo(BaseView view) { return new MyWinExplorerViewInfo(view as MyWinExplorerView); }
        public override BaseViewHandler CreateHandler(BaseView view) { return new MyWinExplorerViewHandler(view as MyWinExplorerView); }
    }
}
